.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - "main"
+      - "release-*"
+
+  pull_request:
+    branches:
+      - "main"
+      - "release-*"
+
+jobs:
+  build:
+    name: Build and Release (${{ matrix.arch }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            arch: linux
+          - os: macos-latest
+            arch: macos
+          - os: windows-latest
+            arch: windows
+      max-parallel: 3
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: "vscode/.nvmrc"
+
+      - name: Install dependencies
+        working-directory: ./vscode
+        run: npm ci
+
+      - name: Lint sources
+        working-directory: ./vscode
+        run: npm run lint
+
+      - name: Build
+        working-directory: ./vscode
+        run: npm run package
+
+      # Install Xvfb and Run tests on Linux
+      - name: Install Xvfb and Run tests (Linux)
+        if: matrix.arch == 'linux'
+        working-directory: ./vscode
+        run: |
+          sudo apt-get install -y xvfb
```

Comment:
it seems `xvfb` is installed on the `ubuntu-latest` - at least it works of us in `ci-repo` workflow.

Reply:
when I tried without that, the npm test failed. IIRC, it was related to some GUI component. I will take a look into the `ci-repo` to see if I am doing something wrong

Reply:
updated it. Thanks for the catch